ghost/milestone-emails/lib/MilestonesEmailService.js

```diff
@@ -2,24 +2,19 @@ const Milestone = require('./Milestone');
 
 /**
  * @template Model
- * @typedef {object} Mention<Model>
+ * @typedef {Object} Mention<Model>
  * @prop {Model[]} data
  */
 
 /**
- * @typedef {object} IMilestoneRepository
```

Comment:
Same here!

Reply:
Not sure I'm following. Are you saying it should be lowercase, or that we're not using the `IMilestoneRepository`, cause I thought we need it for [here](https://github.com/TryGhost/Ghost/pull/16241/files#diff-6b0c2622bbe1fd366bbeb1262f7e2b878284878ad8aea79d15f1b063d24ebe34R27)

Reply:
Oh sorry - I meant about the `object` :)